cart/cart.js

```diff
@@ -0,0 +1,25 @@
+import { renderLine } from './render-line-item.js';
+import cart from '../data/cart.js';
+import games from '../data/games.js';
+import { findById, calcGrandTotal, addDollar } from '../common/utils.js';
+
+
+const tbody = document.querySelector('tbody');
+const grandTotal = document.getElementById('grand-total');
+
+// go through the cart
+for (let i = 0; i < cart.length; i++) {
```

Comment:
It looks like this was copy/pasted from the demo code. Even if you're relying on the demo code for reference, it is important to type the code out yourself.